CONTRIBUTING.md

```diff
@@ -116,6 +116,21 @@ cat > $SUPERCHAIN_REPO/superchain/extra/addresses/$SUPERCHAIN_TARGET/$CHAIN_NAME
 EOF
 ```
 
+#### `extra/genesis-system-configs`
```

Comment:
FYI, I needed to do this to be able to run the op-chain-ops command.

Reply:
makes sense, thanks for fixing!